robot_ws/src/cloudwatch_robot/src/cloudwatch_robot/monitor_obstacle_distance.py

```diff
@@ -20,14 +20,16 @@
 
 import rclpy
 from rclpy.node import Node
+from rclpy.qos import QoSPresetProfiles
 from std_msgs.msg import Header
 from sensor_msgs.msg import LaserScan
 from ros_monitoring_msgs.msg import MetricList, MetricData, MetricDimension
 
 class MonitorObstacleDistance(Node):
     def __init__(self):
         super().__init__('monitor_obstacle_distance')
-        self.scan_sub = self.create_subscription(LaserScan, "/scan", self.report_metric, 5)
+        qos_profile = QoSPresetProfiles.SYSTEM_DEFAULT.value
```

Comment:
I'd note the known issue here as well in case anyone misses the README comment and changes the value.

Reply:
I don't think it needs a comment here as it's not really a workaround, it's just an alternate (probably better) way of setting the QoS policy. If a user changes it they should notice it's not working any more, just like any other code.

Reply:
> I don't think it needs a comment here as it's not really a workaround, it's just an alternate (probably better) way of setting the QoS policy. If a user changes it they should notice it's not working any more, just like any other code.

Will it be as obvious or as quick to diagnose as it has been for us in this case? ;-)

Reply:
as per offline discussion, should use `QoSPresetProfiles.SENSOR_DATA` for sensor data